tox.ini

```diff
@@ -136,7 +136,7 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data
+    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data -W "ignore:The property AsdfFile.blocks has been deprecated*:asdf.exceptions.AsdfDeprecationWarning:asdf.asdf"
```

Comment:
@WilliamJamieson Should we do this to keep our CI from failing until astropy is updated?

Reply:
Nevermind, this worked locally so will require some testing to see why it's not working in the ci.